_data-prepper/pipelines/pipelines.md

```diff
@@ -84,269 +88,8 @@ conditional-routing-sample-pipeline:
         hosts: [ "https://opensearch:9200" ]
         index: all_logs
 ```
+{% include copy-curl.html %}
 
+## Next steps
 
-## Examples
```

Comment:
These are all covered by common use cases pages right?

Reply:
@dlvenable Yes, I deleted these from this file and linked the users to the common use cases.